articles/quickstart/spa/_includes/_authz_protect_client_side_routes.md

```diff
@@ -0,0 +1,3 @@
+## Protect Client-Side Routes
+
+You may want to give access to some routes in your application only to authenticated users. You can check if the user is authenticated with the ${check}.
```

Comment:
My intention here was to use a variable when inserting a snippet (depending on the guide, the method for checking is called differently). Is that the wrong format for entering variables?